compiler/luci-interpreter/pal/linux/pal.cmake

```diff
@@ -68,7 +68,9 @@ macro(add_pal_to_target TGT)
     # instead add sources with visitors in this library
     set(PAL_SOURCES ${TensorFlowSource_DIR}/tensorflow/lite/kernels/internal/tensor_utils.cc
             ${TensorFlowSource_DIR}/tensorflow/lite/kernels/internal/reference/portable_tensor_utils.cc
-            ${TensorFlowSource_DIR}/tensorflow/lite/kernels/internal/quantization_util.cc)
+            ${TensorFlowSource_DIR}/tensorflow/lite/kernels/internal/quantization_util.cc
+            ${TensorFlowSource_DIR}/tensorflow/lite/kernels/kernel_util.cc
+            ${TensorFlowSource_DIR}/tensorflow/lite/c/common.c)
```

Comment:
Q) why are these added?

Reply:
@seanshpark 

when I added the `BroadcastTo` op related code to the luci-interpreter, the following error occurred at the time of build.
```
[ 60%] Linking CXX executable luci_interpreter_memory_manager_test
/usr/bin/ld: libluci_interpreter.so: undefined reference to `tflite::TfLiteTypeGetSize(TfLiteType)'
collect2: error: ld returned 1 exit status
make[2]: *** [compiler/luci-interpreter/src/CMakeFiles/luci_interpreter_memory_manager_test.dir/build.make:124: compiler/luci-interpreter/src/luci_interpreter_memory_manager_test] Error 1
make[1]: *** [CMakeFiles/Makefile2:19323: compiler/luci-interpreter/src/CMakeFiles/luci_interpreter_memory_manager_test.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```
So with the help of @mhs4670go , he asked `BalyshevArtem` about this problem, and I got the following [advice]( https://github.com/Samsung/ONE/pull/11701#issuecomment-1759228267).
Perhaps the version of tensorflow has changed, resulting in the addition of library import.